### PR TITLE
Udpate bigquery package

### PR DESCRIPTION
**Pull request by JakubMatejka**
Solving - https://keboola.atlassian.net/browse/COM-392

### CODE REVIEW

File: test/test_writer_errors.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def test_write_table_sync_error_too_many_values(self, data_dir, credentials_type
             )
             pytest.fail('Must raise exception.')
         except exceptions.UserException as err:
-            assert 'Too many values in row' in str(err)
+            assert 'CSV processing encountered too many errors, giving up' in str(err)
```
Comment (michaljurecko) [inline test/test_writer_errors.py]: Trocha sa obavam, ze zachvilu tu budeme mat dalsi ticket, s tym ze pouzivatel nevie co je zle a dostava tuto chybu.
Nechcem ta zdrzovat na tomto tasku, no co som googlil, tak:
- Job: https://github.com/keboola/google-bigquery-writer/blob/21bfd09f73bc3197871efc18dc2ce93080167f76/google_bigquery_writer/writer.py#L127
- Job by mal mat property `error_result`, pozri [link1](https://stackoverflow.com/a/40853018) a [link2](https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.job.LoadJob.html#google.cloud.bigquery.job.LoadJob.error_result)
- Asi by stalo za to, pozriet sa co to tam vracia.
- Ta chyba sa si vyhadzuje niekde tu: https://github.com/keboola/google-bigquery-writer/blob/21bfd09f73bc3197871efc18dc2ce93080167f76/google_bigquery_writer/writer.py#L197, ... `job` je tam dostupny.

Comment (ondrajodas) [reply test/test_writer_errors.py]: jj máš pravdu, je tam navíc ta zpráva i když errors obsahuje jen tu jednu zprávu, tak vezmeme vždy jen tu první

https://github.com/keboola/google-bigquery-writer/pull/42/files#diff-fb5f437cb09f7ec479a92ef8d19d5fa4R191

Comment (michaljurecko) [reply test/test_writer_errors.py]: jj, `pop` vybera [poslednu](https://www.geeksforgeeks.org/python-list-pop/#:~:text=pop()%20is%20an%20inbuilt,or%20the%20given%20index%20value.&text=Parameter%20%3A,is%20popped%20out%20and%20removed.) spravu :) aj ked premenna sa vola `first...`:

Comment (michaljurecko) [reply test/test_writer_errors.py]: teraz je to ok.

**Review by michaljurecko — APPROVED** [review]
LGTM

**Pull request merged**